tests/guardrails/correction_range_schedule.tests.js

```diff
@@ -2,49 +2,90 @@ const { limits } = require('../../src/index');
 
 var correctionRangeScheduleTests = (test) => {
     var screen;
-    beforeEach(async () => {
+    beforeAll(async () => {
         screen = await test.settingsScreen.OpenCorrectionRangeScreen();
-        await screen.OpenPicker();
-        await screen.SetTime('12:00 AM');
     });
-    afterEach(async () => {
+    afterAll(async () => {
         await screen.Cancel();
     });
-    it('can set at max', async () => {
+    it('can set max units at limit', async () => {
+        await screen.Add();
         await screen.Apply({
-            min: limits.correctionRange.max.limit,
+            min: 100,
             max: limits.correctionRange.max.limit,
         });
+        await screen.AddNewEntry();
+        await expect(screen.GuardrailWarningIconPicker(0)).toBeVisible();
```

Comment:
[nit] What is the number parameter on `GuardrailWarningIconPicker`?

Reply:
yeah maybe a named param would help